Root function

sarcar opened the pull request:
Added example of Quadratic Equation to the section on Functions. I have kept the material on Temperature conversion as well for completeness; during class the instructor can choose to jump over to the Quadratic section is s/he so desires.

sarcar committed:
Add example of Quadratic Roots to teach Functions

sarcar committed:
Changes to Main Menu and Headers to reflect Function instead of Temperature Conversion

graichen commented on Day3/TemperatureConversion.md:
````diff
@@ -44,6 +44,112 @@ Create a new file, temperatureConversion.py. Copy the function definition code a
+Test your program with the following inputs : a = 1, b = -8, c = 15. You should get roots of 5.0 and 3.0 (similar to what is shown below)
+
+```
+C:\Users\305002615\AppData\Local\Continuum\Anaconda3\python.exe C:/SHOURYA/DEV/PYTHON/summerschool/quadroots.py
````
Since we are running in pycharm, let's remove the command line from your output.

sarcar replied:
The commandline is a part of the PyCharm console output , but I can remove it.

graichen commented:
Since the exercises at the bottom are related to the temperature, should they move up
Should we add exercises related to the main program section that build off of both functions?

graichen commented on Day3/TemperatureConversion.md:
```diff
@@ -44,6 +44,112 @@ Create a new file, temperatureConversion.py. Copy the function definition code a
+
+
+
+Test your program with the following inputs : a = 1, b = -8, c = 15. You should get roots of 5.0 and 3.0 (similar to what is shown below)
```
Label this section as an Exercise, so it is easy for students to find the "work".

sarcar replied:
Will do.

sarcar committed:
Minor modifications per review request

sarcar committed:
Added formula for SI, CI in exercise section

graichen commented on Day3/TemperatureConversion.md:
````diff
@@ -170,8 +170,25 @@ if __name__ == "__main__":
+
 5. Write a program that calls two functions ```simpint()``` and ```compint()``` to calculate the Simple Interest and Compound Interest on a (common) Principal Amount, Rate of Interest and Number of Years. Print both the interest values and the difference between them.
 
+   $Simple Interest = \frac{P * R * N}{100}$
````
this does not seem to be the correct formatting for markdown.

sarcar committed:
Changed inline math to images

Inline math is not properly rendered by Github. Changed it to images via codecogs
